config/routes.rb

```diff
@@ -10,10 +10,10 @@
   end
 
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
**[Optional]**

- [ ] In this project, you were supposed to use Gitflow means even for the review. You are merging the development branch into the main branch which is totally violating the requirements of the project. You should have made only one feature branch from the development branch for the whole project and for the review and after getting approval you would merge that feature branch into the development branch. That was the requirement but no worries please be careful next time with the requirements :blush: because when you will work in a company as a professional developer then you will need to be super focused with everything so this habit of having a good focus on requirements would help you a lot in the work environment :+1: